cloud/datasync/example/index.ts

```diff
@@ -35,8 +35,8 @@ const router = middleware.createSyncExpressRouter();
 app.use('/', router);
 
 app.listen(3000, function() {
-  // tslint:disable-next-line:no-console
-  console.log('Example app listening on port 3000!');
+  logger.info('Example app listening on port 3000!',
+    {level: 'INFO', tag: 'cloud:datasync:example', src: 'index.ts'});
```

Comment:
Idea is that parameters you will pass will be serialized. We done investigations for typescript definitions for the most common libraries and they using `params: ...any[]`